### PR TITLE
Made page mobile-friendly, other improvements

### PR DESCRIPTION
**Pull request by codyatwork**
- Made page mobile-friendly
- Fixed numbering
- Fixed typos
- Light accessibility/best practices improvements
- Upgraded dependencies
- Removed unused CSS

### CODE REVIEW

File: _chapters/advanced_strategy.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Since three people are in dispute, it is more difficult to exclude them from the
...
 
 ### Maximising Presidency
-If you are a possible choice for President (i.e. within two clockwise spaces of the initial president since the vote count reset), then it can often make sense to vote down governments to maximise your chance of getting the Presidency and therefore having more of an influence over the game. This is particularly important if a strong power will activated if a Fascist policy is enacted. This is a general strategy that applies to both teams as you can trust yourself more than the others to make the right decision.
+If you are a possible choice for President (i.e. within two clockwise spaces of the initial president since the vote count reset), then it can often make sense to vote down governments to maximise your chance of getting the Presidency and therefore having more of an influence over the game. This is particularly important if a strong power will activate if a Fascist policy is enacted. This is a general strategy that applies to both teams as you can trust yourself more than the others to make the right decision.
```
Comment (7thsanctum) [inline _chapters/advanced_strategy.md]: From my understanding of markdown `+` and `-` just does the same thing. Is there a particular reason for switching it?

File: _chapters/experimental_strategy.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ after: snake
 Early voting can be unproportionally counted on compared to other votes. In the first round of voting, Fascists have the advantage and it is likely that Liberals will vote against, wasting a potential government. Flip-flopping between games can help mitigate this due to balancing skeptical Liberals to optimistic ones. WARNING: This has a side effect of unpredictable player meta; players may see you as untrustworthy because they cannot reason about your initial vote - especially if you were next in line to be President.
 
 ### The Slow Burn
-Liberal investigating a fascist, claims they are a liberal, sits on that information for a while and then revealing that person is a fascist later.
+Liberal, investigating a fascist, claims they are a liberal, sits on that information for a while and then reveals that person is a fascist later.
```
Comment (7thsanctum) [inline _chapters/experimental_strategy.md]: I would change this sentence entirely to something like this:
```suggestion
Here a liberal, investigating a fascist, claims the investigatee is a liberal. They sit on that information for a while and then reveals that the investigated player is a fascist later.
```

File: _chapters/stats_and_probabilities.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ order: 8
 after: snake
 ---
 ### Liberal Draw Odds
-The first government of the policy deck is often the most important and most revealing about a players action. It is only during this point in the game that all players know the number of liberal cards in the deck. From this we can work out the odds of drawing at least one liberal card at these stages. After governments have passed this becomes harder due to the possibilities that players have lied. 
+The first government of the policy deck is often the most important and most revealing about a player's action. It is only during this point in the game that all players know the number of liberal cards in the deck. From this we can work out the odds of drawing at least one liberal card at these stages. After governments have passed this becomes harder due to the possibilities that players have lied.
```
Comment (7thsanctum) [inline _chapters/stats_and_probabilities.md]:
```suggestion
The first government of the policy deck is often the most important and most revealing about a players' action. It is only during this point in the game that all players know the number of liberal cards in the deck. From this we can work out the odds of drawing at least one liberal card at these stages. After governments have passed this becomes harder due to the possibilities that players have lied. 
```

File: _chapters/glossary.md [hunk 4]
```diff
@@ -3,14 +3,67 @@ title: Glossary
...
-Brexit               | Voting down three governments in a row, thus causing a policy to be revealed at random. So-called because some players would vote “no” in order to register dissent, assuming that everyone else would vote yes, but if too many people do this, then the government will actually fail.
-Breakfast            | Another name for Brexit
-Chain Investigation  | The second investigator in a 9/10 player game investigating the first investigator.
-Cut Out              | Someone who is still alive, but is no longer considered for Governments by the table due to them being in a dispute.
-Liberal Block/Corner | A perceived group of Liberal players who are trusted by the majority.
-Power play           | When a player is on the government two rounds in a row.
-Soft Elimination     | When a player is cut out.
-Maverick Cop         | Someone who throws out unsubstantiated claims and their belief cannot be shaken.
-Wheelbarrow          | Someone who is doing something so crazy they should be carted off.
+<div class="responsive-table">
```
[the remark this answers is not in the document]
Comment (codyatwork) [reply _chapters/glossary.md]: It was mostly for the semantics of `<dl>`. Turns out kramdown has its own syntax for that so I have switched the table to that.

File: _chapters/stats_and_probabilities.md (hunk 3 above)
Comment (codyatwork) [reply _chapters/stats_and_probabilities.md]: Do you want to pluralize "player", like, "The first government of the policy deck is often the most important and most revealing about players' actions."? If we're talking about one player, the apostrophe before the S is correct.

Comment (7thsanctum) [reply _chapters/stats_and_probabilities.md]: Yea it's just a single player so your change is right. Had to revisit my understanding of grammar.

**Pull request merged**